web-local/tests/setup/base.ts

```diff
@@ -1,36 +1,41 @@
 import { test as base } from "@playwright/test";
-import { rmSync, writeFileSync, existsSync, mkdirSync } from "fs";
+import { rmSync, existsSync, mkdirSync, cpSync } from "node:fs";
 import { spawn } from "node:child_process";
+import { join } from "node:path";
 import treeKill from "tree-kill";
-import { getOpenPort } from "./getOpenPort";
+import { getOpenPort } from "web-local/tests/utils/getOpenPort";
 import { asyncWaitUntil } from "@rilldata/web-common/lib/waitUtils";
 import axios from "axios";
 
+export const BASE_PROJECT_DIRECTORY = "temp/test-project";
+
+type ProjectInitArgs = { name?: string } | undefined;
 type MyFixtures = {
-  includeRillYaml: boolean;
+  project: ProjectInitArgs;
 };
```

Comment:
Now that there's only one object property here, it looks like this would be cleaner:
```
type MyFixtures = {
  project: string | undefined;
};
```

Reply:
Hmmm I was thinking there might be more config with it. But I guess we could move all of that to `rill.yaml` if possible.